protocol.html

```diff
@@ -717,6 +719,62 @@ <h4 property="schema:name"><span>Note</span>: Conditional Update</h4>
 
                   <p><span about="" id="server-etag" rel="spec:requirement" resource="#server-etag"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> use the HTTP <code>ETag</code> header with a strong validator for RDF bearing representations in order to encourage clients to opt-in to using the <code>If-Match</code> header in their requests.</span></span></p>
                 </div>
+
+                <section id="n3-patch" inlist="" rel="schema:hasPart" resource="#n3-patch">
+                  <h4 property="schema:name">Modifying Resources Using N3 Patches</h4>
+                  <div datatype="rdf:HTML" property="schema:description">
+                    <p><span about="" id="server-patch-n3" rel="spec:requirement" resource="#server-patch-n3"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> support a <code>PATCH</code> request with an <em>N3 Patch</em> body when the target of the request is an <em>RDF document</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>], and indicate such support by listing <code>text/n3</code> as a value of the <code>Accept-Patch</code> header [<cite><a class="bibref" href="#bib-rfc5789">RFC5789</a></cite>] of relevant responses.</span></span> [<a href="https://github.com/solid/specification/issues/125#issuecomment-959518598" rel="cito:citesAsSourceDocument">Source</a>]</p>
+
+                    <p>An <em>N3 Patch</em> is a document in the <em>Notation3 (N3)</em> format [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>], identified by the media type <code>text/n3</code>, conforming to the following structural constraints:</p>
+
+                    <ul>
+                      <li id="server-patch-n3-identifier" rel="spec:requirement" resource="#server-patch-n3-identifier"><span property="spec:statement">The patch document <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain patch resources identified by a URI or blank node <code>?patch</code> which can be chosen freely.</span></li>
+                      <li id="server-patch-n3-type" rel="spec:requirement" resource="#server-patch-n3-identifier">A patch resource <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain a triple [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] <code><span property="spec:statement">?patch rdf:type solid:Patch</span></code>.</li>
```

Comment:
Do you have a pointer to those discussions? I have seen the conclusion (and we have implemented it as such), but I didn't agree at first sight. So would be useful to understand where that came from.

Note that `solid:Patch` was not mandatory in the past (whereas I believe `acl:Authorization` was).

Reply:
Wasn't that the discusssion in #57 ?

Reply:
It was; but I don't see how we've come to the conclusion there, because there were disagreements above.

Reply:
I don't really have a strong opinion on this, but in the interest of progress, will just say that the key to the requirement with `acl:Authorization` is extensibility. That is then balanced against that it takes longer to look up an authorization with the additional triple.

Conceivably, we could have more detailed patch classes in the ontology, so the same could apply there, but OTOH, it seems quite unlikely. My feeling is that the difference part of sameSameButDifferent should prevail, and that a MAY is OK.

Reply:
> will just say that the key to the requirement with acl:Authorization is extensibility

I don't understand how exactly, but happy to drop the discussion for now as to not get distracted.

Reply:
solid:Patch can be subclassed and used to indicate the specialisations such as #server-patch-n3-default or others in the future. I think this is similar to authorizations eg. time-based https://github.com/solid/specification/issues/56#issuecomment-872020854

I think the designs should align here. That aside, as Kjetil, "I don't have a strong open on this". It is worthwhile to record the decision here however it may be.

Reply:
Here is the logic: If you don't demand the `a :Patch` triple, then your hands are tied introducing any other functions which use the same properties as PATCH but have different semantics.  there will be nothing you can do so stop the PATCH engine erroneously taking them as input.  For example suppose we make a sort of `-n` version `DontActuallyPatch` which checks the input but doesn;t  write the data.   The analalogy was adding `TemporaryAuthroization` to teh acl system, would have been impossible were we not insisting that current ACL engines require `Authorisation`.  Is that clear? 
So by the way that means also do not mak e Patch the range of its properties or we will indirectly get the same problem.

Reply:
I follow that argument; should we introduce a subclass then? Like `:InsertWherePatch` or `:SimplePatch`?